app/hotels/src/singleHotel/roomList/formatBeddingInfo.js

```diff
@@ -5,17 +5,12 @@ import idx from 'idx';
 import type { BeddingInfo_room } from './__generated__/BeddingInfo_room.graphql';
 
 export default (room: ?BeddingInfo_room): string => {
-  const type = idx(room, _ => _.type);
   const beddingOptions = idx(room, _ => _.bedding) || [];
-  const bedding = beddingOptions
+  return beddingOptions
     .map(beddingOption => {
       const type = idx(beddingOption, _ => _.type);
       const amount = idx(beddingOption, _ => _.amount);
       return [amount, type].filter(value => value != null).join(' ');
     })
     .join(' or ');
```

Comment:
Not part of the PR, but is this translated later on?

Reply:
It is not and it should be, I will add an todo comment here